src/components/Messages/Messages.tsx

```diff
@@ -56,7 +56,7 @@ const Messages: React.FC<MessagesProps> = ({
   };
   
   // Schedule a scroll after the state update if we're already at the bottom
-  if (isScrolledToBottom()) {
+  if (true || isScrolledToBottom()) {
```

Comment:
why true is needed here ?

Reply:
I think I just turned off the if-branch for testing. Scroll will need to be fixed, this will be changed.